src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -58,16 +62,42 @@ const SampleChatScreen = () => {
     updateChatContinuity
   } = useChatScreenHook(ANONYMOUS);
 
+  const {moveToSignedChannel} = useMoveChatTypeHook();
+
   const updatedChats = updateChatContinuity(chats);
+  const [loading, setLoading] = React.useState(false);
+  const {anonProfileId} = useProfileHook();
+  const ownerChat = selectedChannel?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id === anonProfileId
+  );
+  const memberChat = selectedChannel?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id !== anonProfileId
+  );
 
   const renderChatItem = React.useCallback(({item, index}) => {
     return <BaseChatItem type={ANONYMOUS} item={item} index={index} />;
   }, []);
 
+  const betterSocialMember = selectedChannel?.rawJson?.better_channel_member;
+
   const scrollToEnd = () => {
     flatlistRef.current?.scrollToEnd();
   };
 
+  const moveChatSigned = async () => {
+    try {
+      setLoading(true);
+      await moveToSignedChannel({
+        oldChannelId: selectedChannel?.id,
+        targetUserId: memberChat.user_id
+      });
+    } catch (e) {
+      console.log('error moving chat to signed channel', e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `moveToSignedChannel` function which is wrapped in a try-catch block. This is good for error handling. However, the catch block only logs the error to the console. It would be better to handle the error more gracefully, perhaps by showing an error message to the user.
```diff
  } catch (e) {
-   console.log('error moving chat to signed channel', e);
+   // Show error message to user
+   alert('Error moving chat to signed channel');
  } finally {
    setLoading(false);
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
tidak perlu show alert